export/ExportDialog.cpp

```diff
@@ -200,20 +200,20 @@ void ExportDialog::replaceVars(QString &m, Movie *movie, QDir dir, bool subDir)
     m.replace("{{ MOVIE.LINK }}", QString("movies/%1.html").arg(movie->movieId()));
     m.replace("{{ MOVIE.IMDB_ID }}", movie->id());
     m.replace("{{ MOVIE.TMDB_ID }}", movie->tmdbId());
-    m.replace("{{ MOVIE.TITLE }}", movie->name());
+    m.replace("{{ MOVIE.TITLE }}", movie->name().toHtmlEscaped());
     m.replace("{{ MOVIE.YEAR }}", movie->released().isValid() ? movie->released().toString("yyyy") : "");
-    m.replace("{{ MOVIE.ORIGINAL_TITLE }}", movie->originalName());
-    m.replace("{{ MOVIE.PLOT }}", movie->overview().replace("\n", "<br />"));
-    m.replace("{{ MOVIE.PLOT_SIMPLE }}", movie->outline().replace("\n", "<br />"));
-    m.replace("{{ MOVIE.SET }}", movie->set());
-    m.replace("{{ MOVIE.TAGLINE }}", movie->tagline());
-    m.replace("{{ MOVIE.GENRES }}", movie->genres().join(", "));
-    m.replace("{{ MOVIE.COUNTRIES }}", movie->countries().join(", "));
-    m.replace("{{ MOVIE.STUDIOS }}", movie->studios().join(", "));
-    m.replace("{{ MOVIE.TAGS }}", movie->tags().join(", "));
-    m.replace("{{ MOVIE.WRITER }}", movie->writer());
-    m.replace("{{ MOVIE.DIRECTOR }}", movie->director());
-    m.replace("{{ MOVIE.CERTIFICATION }}", movie->certification());
+    m.replace("{{ MOVIE.ORIGINAL_TITLE }}", movie->originalName().toHtmlEscaped());
+    m.replace("{{ MOVIE.PLOT }}", movie->overview().toHtmlEscaped().replace("\n", "<br />"));
+    m.replace("{{ MOVIE.PLOT_SIMPLE }}", movie->outline().toHtmlEscaped().replace("\n", "<br />"));
+    m.replace("{{ MOVIE.SET }}", movie->set().toHtmlEscaped());
+    m.replace("{{ MOVIE.TAGLINE }}", movie->tagline().toHtmlEscaped());
+    m.replace("{{ MOVIE.GENRES }}", movie->genres().join(", ")).toHtmlEscaped();
```

Comment:
Should be `m.replace("{{ MOVIE.GENRES }}", movie->genres().join(", ").toHtmlEscaped());` :smiley: 
GCC says: `export/ExportDialog.cpp:210:80: warning: ignoring return value of ‘QString QString::toHtmlEscaped() const’, declared with attribute nodiscard [-Wunused-result]`
Will update this later.

Reply:
Good catch, thanks!

Reply:
No problem.  :smile: 
It's fixed by #397